RecoMET/METPUSubtraction/python/mvaPFMET_leptons_cfi.py

```diff
@@ -99,21 +99,24 @@
     filter = cms.bool(False)
     )
 
-isomuonseq     = cms.Sequence(isomuons)
-isoelectronseq = cms.Sequence(isoelectrons)
-isotauseq      = cms.Sequence(
-    hpsPFTauDiscriminationByLooseCombinedIsolationDBSumPtCorr3Hits*
-     #kt6PFJetsForRhoComputationVoronoiMet*
-     #hpsPFTauDiscriminationByMVAIsolation*
-     hpsPFTauDiscriminationAgainstMuon2*
+isomuonTask     = cms.Task(isomuons)
+isomuonseq      = cms.Sequence(isomuonsTask)
+isoelectronTask = cms.Task(isoelectrons)
+isoelectronseq  = cms.Sequence(isoelectronsTask)
+isotauTask      = cms.Task(
+     hpsPFTauDiscriminationByLooseCombinedIsolationDBSumPtCorr3Hits,
+     #kt6PFJetsForRhoComputationVoronoiMet,
+     #hpsPFTauDiscriminationByMVAIsolation,
+     hpsPFTauDiscriminationAgainstMuon2,
      isotaus
     )
+isotauseq      = cms.Sequence(isotauTask)
 
 leptonSelection = cms.PSet(
```

Comment:
@camclean @alefisico this `leptonselection` doesn't seem to be used anywhere
What is its purpose? If still needed, can the sequences listed in `SelectEvents` be replaced by the corresponding tasks? If so, one could get rid of the `isoleptonseq`'s here, with some cleaning

Reply:
Maybe MET conveners can comment on this? @alkaloge @mcremone

Reply:
Ok, probably this is not a relevant enough issue worth to keep this otherwise concluded PR in standby.

Anyhow, most of the configs modified here seems already not being in use anywhere in CMSSW. The same also applies for the `leptonSelection` addressed earlier in this comment: although it could be nice to clean it up if not needed, as well as the iso lepton sequences in it, they were already there before this PR, and we can limit it to the technical move from sequences to tasks, which is the goal of this update.

I'm going to sign this PR for reco, and I let MET conveners (@alkaloge @mcremone) the task of reviewing and finally cleaning the MET configurations available (and maintained...) in CMSSW